src/reducers/index.js

```diff
@@ -57,9 +57,27 @@ export const removeResource = (state, action) => {
   return validate(newState)
 }
 
+// shameless green -- nearly duplicates setRetrieveError()
```

Comment:
I don't know what this means.

Reply:
hrmm.  Jeremy didn't like the longer comment;  this is too terse for you.  Lemme try a slight revision.

Reply:
this is from the Sandi Metz class -- implementing something that works, but that isn't necessarily optimal code, that passes tests ... is `shameless green`.  This phrase appears in various code base comments.

The bit after explains how this may ultimately be suboptimal code.